INSTALL.md

```diff
@@ -3,6 +3,16 @@
 [CMake](https://cmake.org/) and [Bazel](https://bazel.build) are the official
 build systems for opentelemetry-cpp.
 
+## Dependencies
+
```

Comment:
There is also a more detailed dependencies doc here ( https://github.com/open-telemetry/opentelemetry-cpp/blob/main/docs/dependencies.md).
Can we add a reference to the above doc along with https://github.com/open-telemetry/opentelemetry-cpp/blob/main/third_party_release for versions in this section?

Reply:
oh, how I missed this file :). 
Thanks removed the libs.